dart.yaml

```diff
@@ -73,9 +79,10 @@ update:
     - "analyzer-*"
     - "meta-*"
     - "merge*"
-  github:
-    identifier: dart-lang/sdk
-    use-tag: true
+  git: {}
+  schedule:
+    period: monthly
```

Comment:
We may want to make this weekly? I worry that a monthly cadence would make us very slow to pick up new versions, since you could have up to a 30 day lag time before a re-poll is done.